docs/source/kale.pipeline.rst

```diff
@@ -30,6 +30,14 @@ kale.pipeline.domain\_adapter module
    :undoc-members:
    :show-inheritance:
 
+kale.pipeline.mogonet\_multiomics\_trainer module
```

Comment:
Pipeline should be as reusable as possible so needs to be high-level. Can we think of a better, more generic name that can cover multiple models following the same workflow? Would your method be called also mogonet? We cannot affort to have one pipeline per algorithm.